<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>371</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLineEdit" name="line4"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="line8"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="line7"/>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="line1"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="line5"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line2"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="line3"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="line6"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="line9"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Последнее появившееся слово:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Answer">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Start">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_alert">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_openFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
